whitesands/code/modules/mining/lavaland/ash_flora.dm

```diff
@@ -145,13 +145,12 @@
 
 /obj/item/seeds/lavaland/whitesands/puce/attackby(obj/item/item, mob/user, params)
 	. = ..()
-	if(!istype(item ,/obj/item/kitchen/knife))
+	if(!istype(item, /obj/item/kitchen/knife))
 		return
-	playsound(loc, 'sound/effects/glassbr1.ogg', 50, TRUE, -1)
-	to_chat(user, "<span class='notice'>You start breaking the [src] up into shards...")
-	if(!do_after(user, 10, src))
+	playsound(src, 'sound/effects/glassbr1.ogg', 50, TRUE, -1)
+	to_chat(user, "<span class='notice'>You start breaking the [src] up into shards...</span>")
+	if(!do_after(user, 1 SECONDS, src))
 		return
-	to_chat(user, "<span class='notice'>You finish breaking the [src]")
-	var/location = get_turf(src)
-	new /obj/item/garnish/puce(location)
+	to_chat(user, "<span class='notice'>You finish breaking the [src]</span>")
+	new /obj/item/garnish/puce(get_turf(src))
```

Comment:
as its an item, you can cut it in hand. i think youll want to put it in drop_location and put in hand if possible?

Reply:
for the other ones too

Reply:
still has to be done btw